### ISSUE TITLE
Exception stack trace does not include inner exception data

### ISSUE THREAD

**Issue by DonMcNamara** [Iristyle/NLog.AirBrake#24]
I just noticed that the exception stack trace (back trace) only includes the data from the top most exception.

Should we include the stack trace of the inner exceptions as well?

**Comment by Iristyle** [comment]
Yes... as many frames as possible I would say... maybe a configurable limit with a sensible default... like 20?

**Comment by Elindalyne** [comment]
I can take care of this for you guys.
Do we want 20 lines to be the limit per exception or 20 lines total?

**Comment by Iristyle** [comment]
Yeah, go to town!
